sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/models.py

```diff
@@ -232,10 +281,28 @@ def __init__(
     def _from_deleted_key_bundle(cls, deleted_key_bundle):
         # type: (_models.DeletedKeyBundle) -> DeletedKey
         """Construct a DeletedKey from an autorest-generated DeletedKeyBundle"""
+        jwk = {
+            "kid": deleted_key_bundle.key.kid,
+            "kty": deleted_key_bundle.key.kty,
+            "key_ops": deleted_key_bundle.key.key_ops,
+            "n": deleted_key_bundle.key.n,
+            "e": deleted_key_bundle.key.e,
+            "d": deleted_key_bundle.key.d,
+            "dp": deleted_key_bundle.key.dp,
+            "dq": deleted_key_bundle.key.dq,
+            "qi": deleted_key_bundle.key.qi,
+            "p": deleted_key_bundle.key.p,
+            "q": deleted_key_bundle.key.q,
+            "k": deleted_key_bundle.key.k,
+            "t": deleted_key_bundle.key.t,
+            "crv": deleted_key_bundle.key.crv,
+            "x": deleted_key_bundle.key.x,
+            "y": deleted_key_bundle.key.y,
+        }
```

Comment:
This complicated dict literal is identical to the other one above, maybe condense to a helper function that is shared in both places:
```
fields = ["kid", ...]

def jwk_from_key_bundle(key_bundle):
   return { f: getattr(key_bundle.key, f) for f in fields } 
```

Then that function and the contents of fields are easy to put under unit test.